src/help/HelpCommandHandlers.js

```diff
@@ -39,7 +39,8 @@ define(function (require, exports, module) {
         StringUtils             = require("utils/StringUtils"),
         UpdateNotification      = require("utils/UpdateNotification"),
         AboutDialogTemplate     = require("text!htmlContent/about-dialog.html"),
-        ContributorsTemplate    = require("text!htmlContent/contributors-list.html");
+        ContributorsTemplate    = require("text!htmlContent/contributors-list.html"),
+        HealthDataNotification  = require("healthData/HealthDataNotification");
```

Comment:
Nit: move this above the `text!` requires.  (Though actually, this change will go away if we move Health Data into a core extension...)